docs/api/tokens/fungible-token.schema.json

```
@@ -0,0 +1,25 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "fungible-token-metadata",
+  "title": "FTMetadataResponse",
```

I would suggest we follow full naming conventions here. We can later create an alias type to a shortened name

```suggestion
  "title": "FungibleTokenMetadataResponse",
```